Brancybrancyplayground

catzkorn opened the pull request.

Commit:
Updated database setup file.

Including date_availability collumn.

Commit:
Updated database setup file.

Including date_availability collumn.

Commit:
Created feature test for viewing dates.

Updates see user spaces with a feature test for viewing dates.

Commit:
Created feature test for viewing dates.

Updates see user spaces with a feature test for viewing dates.

Commit:
added the capybara function date-menu

used a html <select> tag to view the dates

Commit:
added the capybara function date-menu

used a html <select> tag to view the dates

Commit:
added date menu

Commit:
added date menu

CRTW26 committed:
Merge branch 'master' into bookingsystem
merging master and bookingsystem

CRTW26 committed:
Adds date attribute to Space

adds method and testing for date in Space class

CRTW26 committed:
adding date functionality to space manager

Commit:
Merge branch 'bookingsystem' of github.com:Catzkorn/makersbnb into bookingsystem

Commit:
Adding array of dates to the database.

Setting up date availability in database.

Commit:
End of day push.

Commit:
Created bookings table schema.

Updated database_setup.sql to include new schema for bookings.

Commit:
Refactored doubles into tests.

Refactored the tests to use doubles.

Commit:
Created Booking class and BookingManagement.

Bookingmanagment has a request booking function that puts a booking into the database.

Commit:
Fixed SQLInjection issue with date array.

Added back  and added the join method into the array.

Commit:
Added booking availability.

Method to show availability per month.

Commit:
Merged from master.

Merged in master.

0xfarhaan reviewed (approved):
All good to go feel free to merge

Commit:
added a new view space for each place

Commit:
Removed error from space/:spaceid.

Removed error causing rackup testing to fail.

Commit:
Added view specific space spec

feature test for vew specific spaces

Commit:
Able to view all text dates available.

an erb shows all avaible dates for a space.

Commit:
Updated tests,

Updated tests for the new methods.

Commit:
FRefactored date viewing page.

Small refactor.

Commit:
Added a request booking button.

Booking request method needs to be updated to only removed bookings that have been confirmed by hte host.
